geonode_mapstore_client/client/js/components/home/FeaturedList.jsx

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2020, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { Spinner, Button } from 'react-bootstrap-v1';
+import HTML from '@mapstore/framework/components/I18N/HTML';
+import ResourceCard from '@js/components/home/ResourceCard';
+import FaIcon from '@js/components/home/FaIcon';
+import { withResizeDetector } from 'react-resize-detector';
+
+const Cards = withResizeDetector(({
```

Comment:
The width seems provided by the parent so the withResizeDetector is not needed here. Please check if removing this has some effect